extensions/amp-form/amp-form.md

```diff
@@ -196,6 +201,17 @@ You can configure your analytics to send these events as in the example below.
 </amp-analytics>
 ```
 
+`amp-form-submit` is fired before a request is sent. For this event, a set of variables that reflects your form's fields is generated. For an example, consider the following form:
```

Comment:
Consider rewriting the text (above and below) for clarity and formatting:

The `amp-form-submit` event fires when a form request is initiated.  The `amp-form-submit` event generates a set of variables that correspond to the specific form and the fields in the form.  These variables can be used for analytics.

For example, the following form has two fields:

`code snippet...`


When the `amp-form-submit` event fires, it generates the following variables containing the values that were specified in the form: 

-  `formId`
- `formFields[user_name]`
- `formFields[user_password]`